.build/docker/run-tests.sh

```diff
@@ -178,7 +208,7 @@ TEST_SCRIPT=${test_script}
 JAVA_VERSION=${java_version}
 PYTHON_VERSION=${python_version}
 cython=${cython}
-ANT_OPTS="-Dtesttag.extra=.arch=$(arch).python${python_version}"
+ANT_OPTS="${ANT_OPTS}"
```

Comment:
why ?

Reply:
Oh, indeed. Must be a left-over from an experiment. Fix in a moment.